cartridges/notify_me/cartridge/client/default/js/modal/modal.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+/**
+ * Generates the modal window on the first call.
+ *
+ */
+function getModalHtmlElement() {
+    if ($("#notifyMeModal").length !== 0) {
+        $("#notifyMeModal").remove();
+    }
+    var htmlString =
+        "<!-- Modal -->" +
+        '<div class="modal fade" id="notifyMeModal" tabindex="-1" role="dialog">' +
+        '<span class="enter-message sr-only" ></span>' +
+        '<div class="modal-dialog quick-view-dialog">' +
+        "<!-- Modal content-->" +
+        '<div class="modal-content">' +
+        '<div class="modal-header">' +
+        '    <button type="button" class="close pull-right" data-dismiss="modal">' +
+        '        <span aria-hidden="true">&times;</span>' +
+        '        <span class="sr-only"> </span>' +
+        "    </button>" +
+        "</div>" +
+        '<div class="modal-body"></div>' +
+        '<div class="modal-footer"></div>' +
+        "</div>" +
+        "</div>" +
+        "</div>";
+    $("body").append(htmlString);
+}
+
+/**
+ * replaces the content in the modal window for product variation to be edited.
+ * @param {string} url - url to be used to retrieve a form html content
+ */
+function fillModalElement(url) {
+    $("#notifyMeModal").spinner().start();
```

Comment:
Too much use of the same jQuery element search.
Better to define the first search result into a variable and use it instead. Like
`
$notifyMeModal = $("#notifyMeModal");
// and then use this variable
// in case of searching in the DOM variable we can use a 'find' method
$notifyMeModal.find('.modal-body');
`